modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -790,6 +813,32 @@ TEST_F(ONNXMediaFrameTest, InferListYUV)
     // Validate
     validate();
 }
+TEST_F(ONNXWithRemap, InferWithDisabledOut)
+{
+    useModel("object_detection_segmentation/faster-rcnn/model/FasterRCNN-10");
+    // Create tensor without batch
+    cv::Mat rand_mat = initMatrixRandU(CV_32FC3, cv::Size{800, 800});
+    std::vector<int> dims = {rand_mat.channels(), rand_mat.rows, rand_mat.cols};
+    cv::Mat in_mat(dims, CV_32F, rand_mat.data);
+    // ONNX_API code
+    infer<float>(in_mat, out_onnx, {"6379", "6383"});
+    // G_API code
+    using FRCNNOUT = std::tuple<cv::GMat, cv::GMat>;
+    G_API_NET(FasterRCNN, <FRCNNOUT(cv::GMat)>, "FasterRCNN");
+    auto net = cv::gapi::onnx::Params<FasterRCNN>{model_path}
+        .cfgOutputLayers({"out1", "out2"})
+        .cfgPostProc({cv::GMatDesc{CV_32F, {1,20}},
+                      cv::GMatDesc{CV_32F, {5}}}, remapRCNNPorts, {"6383", "6379"});
```

Comment:
Can the disable output feature works without remap function ? I would reflect it in tests.

Reply:
No. `disable output` works for dynamic outputs case. Nets with static outputs don't use remap function. But if you try use net with dynamic outputs then you receive error message about it.